runner/runner.go

```diff
@@ -249,8 +253,7 @@ func (runner *Runner) StopInstance(instanceID string) (err error) {
  **********************************************************************************************************************/
 
 func (runner *Runner) monitorUnitStates() {
-	statusChan, errChan := runner.systemd.SubscribeUnitsCustom(
```

Comment:
`runner.SubscribeUnitsCustom` returns map (where key is unit name) of changed systems units in status channel. I that in case service stopped this map contains `nil` value for this unit. Thu, no need to implement own subscription method but better to handle this particular case.

Reply:
Done